src/cargo-fmt/main.rs

```diff
@@ -90,7 +94,27 @@ fn execute() -> i32 {
 
     let strategy = CargoFmtStrategy::from_opts(&opts);
 
-    handle_command_status(format_crate(verbosity, &strategy, opts.rustfmt_options))
+    if opts.manifest_path.is_some() {
+        let specified_manifest_path = opts.manifest_path.unwrap();
```

Comment:
Could you replace these two lines with `if let Some() = `?

```rust
if let Some(specified_manifest_path) = opts.manifest_path {
```

Reply:
Done!